src/index.ts

```diff
@@ -2,6 +2,14 @@ import express from 'express';
 import config from './config';
 import loaders from './loaders';
 
+process.on('uncaughtException', err => {
+  throw new Error(err.message);
```

Comment:
Add logging. What's a point of catching it and throwing. The same behaviour if you delete it?

Reply:
added log instear of error throw